lib/DDG/Spice/DoiBibtex.pm

```diff
@@ -0,0 +1,25 @@
+package DDG::Spice::DoiBibtex;
+
+use DDG::Spice;
+
+description "Look up a digital object identifier (bibtex code)";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/DoiBibtex.pm";
+attribution github => ["https://github.com/nomeata", "Joachim Breitner"],
+            web => ["http://www.joachim-breitner.de", "Joachim Breitner"],
+            email => ['mail@joachim-breitner.de', "Joachim Breitner"];
+status "enabled";
+
+triggers start => 'This spice should never match any request, as it is just a helper for the doi spice.';
```

Comment:
@jbarrett we now have a much simpler way for your to create this now!

http://docs.duckduckhack.com/backend-reference/api-reference.html (checkout the "Multiple API Endpoints" section) -- you'll want to use the `spice alt_to` feature